### ISSUE TITLE
[Enhancement] Autostart Executable Functionality

### ISSUE THREAD

**Issue by NaiveInvestigator** [datasone/MPVMediaControl#6]
Continuation of this thread.
#3

**Comment by NaiveInvestigator** [comment]
@datasone I have an idea but not so sure how to implement this. We can remedy the problem with multiple instances by using a mutex, which will only allow only one instance of MPVControlMedia.exe to run. And as for when to close it I still have to figure out that one.

**Comment by datasone** [comment]
Fixed in [0.1.0](https://github.com/datasone/MPVMediaControl/releases/tag/0.1.0)

**Comment by NaiveInvestigator** [comment]
@datasone could we add instructions to put in executable inside the folder and rename the script to main.lua. 
Like this:

![image](https://user-images.githubusercontent.com/72654596/158016197-59c94465-e367-4e93-a581-990c958a270e.png)

The reason for doing this is because so that ppl don't have to worry where to put the executable, will make the installation more friendly for new ppl and the directory of the executable won't be hard coded. Instead we can use `mp.get_script_directory()` to get the directory of the script and just add "/MPVMediaControl.exe". That will reduce problems imo.

```
script_dir = mp.get_script_directory()
MPVMC_PATH = script_dir .. "/MPVMediaControl.exe"
```

Adding these two lines should do the trick if you're feeling lazy lol. 


Thank you so much for attending my TED talk :> jk thanks for the updates, works like a charm now!! Was really excited for them lol.

P.S :  Could we add this script in:
https://github.com/mpv-player/mpv/wiki/User-Scripts

You know so ppl can find it and use it too.
